server/main.lua

```diff
@@ -447,15 +447,15 @@ end)
 QBCore.Functions.CreateUseableItem("handcuffs", function(source, item)
     local src = source
     local Player = QBCore.Functions.GetPlayer(src)
-    if Player.Functions.GetItemByName(item) then
+    if Player.Functions.GetItemByName('handcuffs') then
         TriggerClientEvent("police:client:CuffPlayerSoft", src)
     end
 end)
 
 QBCore.Functions.CreateUseableItem("moneybag", function(source, item)
     local src = source
     local Player = QBCore.Functions.GetPlayer(src)
-    if Player.Functions.GetItemByName(item) then
```

Comment:
Because CreateUsableItem return a table the solution here is just change this Player.Functions.GetItemByName(item) to this Player.Functions.GetItemByName(item.name)